src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public DifferentColors getRandomIndex() {
+        int index = new Random().nextInt(DifferentColors.values().length);
+        return DifferentColors.values()[index];
```

Comment:
```suggestion
    private final Random random = new Random();
    
    public DifferentColors getRandomIndex() {
        int index = random.nextInt(DifferentColors.values().length);
        return DifferentColors.values()[index];
```
Random should be a class field

Reply:
Why?